precompiles/gov/events_test.go

```diff
@@ -165,3 +168,146 @@ func (s *PrecompileTestSuite) TestVoteWeightedEvent() {
 		})
 	}
 }
+
+func (s *PrecompileTestSuite) TestDepositEvent() {
+	var (
+		stDB   *statedb.StateDB
+		ctx    sdk.Context
+		method = s.precompile.Methods[gov.DepositMethod]
+	)
+
+	testCases := []struct {
+		name        string
+		malleate    func(depositor common.Address, proposalId uint64, amount sdk.Coins) []interface{}
+		postCheck   func()
+		gas         uint64
+		expError    bool
+		errContains string
+	}{
+		{
+			"success - the correct Deposit event is emitted",
+			func(depositor common.Address, proposalId uint64, amount sdk.Coins) []interface{} {
+				return []interface{}{
+					depositor,
+					proposalId,
+					cmn.NewCoinsResponse(amount),
+				}
+			},
+			func() {
+				log := stDB.Logs()[0]
+				s.Require().Equal(log.Address, s.precompile.Address())
+
+				// Check event signature matches the one emitted
+				event := s.precompile.ABI.Events[gov.EventTypeDeposit]
+				s.Require().Equal(crypto.Keccak256Hash([]byte(event.Sig)), common.HexToHash(log.Topics[0].Hex()))
+				s.Require().Equal(log.BlockNumber, uint64(ctx.BlockHeight())) //nolint:gosec // G115
```

Comment:
nit: expected is the first argument:
```suggestion
				s.Require().Equal(uint64(ctx.BlockHeight()), log.BlockNumber) //nolint:gosec // G115
```